server/pyproject.toml

```diff
@@ -14,6 +14,7 @@ psycopg = {extras = ["binary", "pool"], version = "^3.1.19"}
 freezegun = "^1.5.1"
 gunicorn = "^22.0.0"
 python-dotenv = "^1.0.1"
+django-extensions = "^3.2.3"
```

Comment:
whats this for?

Reply:
It gives the reset_db command. flush only clears the data and doesn't reset migrations